ntp.go

```diff
@@ -38,6 +38,15 @@ func (t ntpTime) UTC() time.Time {
 	return time.Date(1900, 1, 1, 0, 0, 0, 0, time.UTC).Add(time.Duration(nsec))
 }
 
+func toNtpTime(t time.Time) ntpTime {
+	seconds := uint32(t.Unix() - time.Date(1900, 1, 1, 0, 0, 0, 0, time.UTC).Unix())
```

Comment:
Might be clearer to do something like https://github.com/ntp-project/ntp/blob/master/util/jitter.c.

That is, define a const JAN_1970 which is number of seconds since ntp epoch (`2208988800` seconds)